pkg/relayer/proxy/proxy_test.go

```diff
@@ -658,12 +658,12 @@ func sendRequestWithDifferentSession(
 	t *testing.T,
 	test *testproxy.TestBehavior,
 ) (errCode int32, errorMessage string) {
-	// Use secondaryService instead of service1 so the session IDs don't match
+	// Use a block height that generates a different session ID
 	req := testproxy.GenerateRelayRequest(
 		test,
 		appPrivateKey,
-		secondaryService,
-		blockHeight,
+		defaultService,
+		blockHeight+sessionkeeper.GetSessionGracePeriodBlockCount(),
```

Comment:
Can you declare a local height variable right below the comment so it's easier to read & follow.?